3.6/oasis/backups.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+description: Quick guide about backups in Oasis for ArangoDB.
+title: Backups in Oasis
+---
+
+# How to create backups
+
+To backup data in Oasis for an ArangoDB installation, navigate to the backup section of your deployment created previously.
+
+![Backup ArangoDB](../images/oasis-backup-section.png)
+
+There are two types of backups. Periodic backups taken through backup policies and manual backups taken by hand.
+
+## Periodic backups
+
+Periodic backups are taken on a given timeframe via a Backup Policy. To see when the new backup is due, observe the schedule section.
+
+![Backup Policy schedule](../images/oasis-backup-policy-schedule.png)
+
+Upon creating a deployment, by default, a new policy is created which will take backups of a deployment hourly every two hours. To edit this policy (or any policy), highlight it in the row above and hit the pencil icon.
+
+![Edit Backup Policy](../images/oasis-edit-backup-policy.png)
+
+These backups are not automatically uploaded.
+
+## Manual backups
+
+It's also possible to create a backup on demand. To do this, click **Back up now**.
+
+![Back up Now](../images/oasis-back-up-now.png)
+
+![Back up Now Dialog](../images/oasis-back-up-now-dialog.png)
+
+## Uploading backups
+
+By default a backup is not uploaded to the cloud, meaning it's ephemeral. It will go away and not persist (by default in 6 hours). To keep a backup for a longer period of time, upload it to the cloud storage. Uploaded backups are required for [cloning](#how-to-clone-a-deployment-using-a-backup).
+
+# Restoring from backup
+
+To restore a database from a backup, highlight the desired backup and click on the restore icon.
```

Comment:
Add warning that:

1) All current data will be lost
2) During restore the deployment is temporarily not available